### PR TITLE
Build with dotnet

### PR DESCRIPTION
**Pull request by victorhurdugaci**
cc @Tratcher

### CODE REVIEW

File: test/TestSites/project.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "content": [
+    "wwwroot/**/*",
```
Comment (moozzyk) [inline test/TestSites/project.json]: Isn't just `wwwroot` enough?

**Comment by moozzyk** [comment]
:ship::it:

File: test/TestSites/wwwroot/web.config [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <configuration>
   <system.webServer>
-    <httpPlatform forwardWindowsAuthToken="true" />
```
Comment (Tratcher) [inline test/TestSites/wwwroot/web.config]: Please comment that there's a tooling issue that prevents us from using publish-iis in the same repo as it's src.

File: test/TestSites/project.json (hunk 1 above)
Comment (GuardRex) [inline test/TestSites/project.json]: Yes, `wwwroot` is working here to get everything in the folder published.

**Comment by victorhurdugaci** [comment]
Updated

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**